packages/interactive_media_ads/README.md

```diff
@@ -240,20 +249,43 @@ Future<void> _requestAds(AdDisplayContainer container) {
     },
   );
 
-  return _adsLoader.requestAds(AdsRequest(adTagUrl: _adTagUrl));
+  return _adsLoader.requestAds(AdsRequest(
+    adTagUrl: _adTagUrl,
+    contentProgressProvider: _contentProgressProvider,
+  ));
 }
 
-Future<void> _resumeContent() {
+Future<void> _resumeContent() async {
   setState(() {
     _shouldShowContentVideo = true;
   });
-  return _contentVideoController.play();
+
+  if (_adsManager != null) {
+    _contentProgressTimer = Timer.periodic(
+      const Duration(milliseconds: 500),
```

Comment:
@harold1208 @stuartmorgan What are your thoughts on the interval to be used here in the example? This works for me on Android and iOS, but this is a Flutter specific workaround to handle the SDK running on a separate thread.

Reply:
For context, this timer updates the SDK of the progress of the content video with this being the interval between updates.

Reply:
Per our offline discussion with the IMA team, a 200ms interval would be the ideal value.